extensions/guacamole-auth-json/src/main/resources/translations/tr.json

```diff
@@ -0,0 +1,7 @@
+{
+
+    "DATA_SOURCE_JSON" : {
+        "NAME" : "JSON"
```

Comment:
If this string is not being translated, you don't need to provide this - Guacamole will fall back to the English translation. If "JSON" is the Turkish translation of the English "JSON", then please leave it.